deploy/lambda/whitelist.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import botocore
+from botocore.vendored import requests
```

Comment:
either import specific submodules or just import the global `botocore` parent module. inconsistent to use both